### DISCUSSION TITLE
How to change font in Gatsby?

### DISCUSSION THREAD

**Discussion by mimoid-prog** [chakra-ui/chakra-ui#3734] [Q&A]
I was following this article [https://chakra-ui.com/guides/using-fonts](https://chakra-ui.com/guides/using-fonts) but there is no info how to change font in Gatsby, only in CRA and Next.js. Where should be Fontsource fonts imported to make this work?

**Comment by ayuhito** [comment]
It's fairly flexible. You could choose to import it into a layout template (layout.js), page component (index.js), or gatsby-browser.js. You control where it should load.
